jupyter-on-gke/variables.tf

```diff
@@ -70,4 +70,10 @@ variable "url_domain_name" {
   type = string
   description = "Name of the domain provided by the user. This var will only be used if url_domain_addr is not empty"
   default = ""
+}
+
+variable "support_email" {
+  type = string
+  description = "Email displayed on the OAuth consent screen."
```

Comment:
Maybe we can add a Pantheon link to where this email can be found?

Reply:
Oh wait, this is part of the automatically creating clients, required field for creating a brand. Let me know if you would like me to create another PR for this. 

But without a brand, they won't see the email on the Pantheon page.

Reply:
I'll add a better description, not exactly telling you what it is.